### PR TITLE
allow multiple to in sms action

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue: https://github.com/telefonicaid/perseo-fe/issues/337
Current implementation uses whitespace as split number separator( i.e.: `+346666666 +347777777 +359999999`)
- [x] Backward compatible
- [x] Tested
- [x] Update Doc
- [x] Update tests

### CODE REVIEW

**Comment by AlvaroVega** [comment]
Tested with a visual rule including 2 numbers in SMS action:
```
iot-perseo-fe               | time=2019-05-30T07:06:03.678Z | lvl=INFO | corr=5ea560c1-0370-43e9-913a-5452780e4740; perseocep=3 | trans=e472fe39-6705-40aa-b2d0-479d5c273dea | op=/actions/do | path=/actions/do | comp=perseo-fe | srv=smartcity | subsrv=/test | msg=smsAction.SendSMS done url:http://sms-endpoint/smsoutbound result:{"errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","hostname":"sms-endpoint","host":"sms-endpoint","port":80}, msg: {"to":["tel:+3412341234","tel:+346656566"],"message":"asdfasdfasdfasdf","from":"tel:22012;phone-context=+34"}
```

**Comment by AlvaroVega** [comment]
Check if ACDC support https://github.com/telefonicaid/perseo-fe/issues/337#issuecomment-495513403 multiple sms destinations format as `msg: {"to":["tel:+3412341234","tel:+346656566"],"message":"asdfasdfasdfasdf","from":"tel:22012;phone-context=+34"}`

File: test/acceptance/integration/Rules/EPL/epl_create_rules.feature [hunk 1]
```diff
@@ -71,6 +71,21 @@ Feature: Append a new rule in Perseo manager
     Then I receive an "200" http code in rules request
     And Validate that rule name is created successfully in db
 
+  @happy_path
+  Scenario: Append a sms multiple rule in Perseo manager
```
Comment (fgalan) [inline test/acceptance/integration/Rules/EPL/epl_create_rules.feature]: Apart from this acceptance test (which I'm not sure is in use now), I'd suggest adding a component test. You can base in existing "should return ok with a valid action with a rule for sms", just adding a new case (a new JSON) which the new syntaxis for multi-SMS.

**Comment by fgalan** [comment]
We are going to freeze version and it would be a good idea to include this feature before doing it. Thus, we are going to merge this, leaving the pending test described at https://github.com/telefonicaid/perseo-fe/pull/341#discussion_r288941035 for the future (issue #337 will keep opened)

**Pull request merged**